.github/workflows/sync.yml

```
@@ -118,57 +124,102 @@ jobs:
-        uses: peter-evans/create-pull-request@v5
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          branch: sync/hotfix-to-develop
-          base: develop
-          title: "🔄 Sync: Hotfix from main to develop"
-          body: |
-            # ⚠️ Sincronización de hotfix (ALTA PRIORIDAD)
+        run: |
+          echo "Creando rama para el PR manualmente..."
+          SYNC_BRANCH="sync/hotfix-to-develop"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)